SimG4CMS/Calo/test/python/runEta13_cfg.py

```diff
@@ -5,7 +5,7 @@
 process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi")
 process.load("Geometry.CMSCommonData.cmsSimIdealGeometryXML_cfi")
```

Comment:
@bsunanda - in general, it is recommended to use geometry from DB to assure reproducibility and avoid mixing runs. Both are defined by GT in this case.